### DISCUSSION TITLE
Browser Support

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/spmssite#35] [Q&A] [answered]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/gbowne1/spmssite/blob/main/.github/CONTRIBUTING.md)
- [X] I agree to follow the [Code of Conduct]().
- [X] I have searched for [existing issues]() that already include this feature request, without success.

### Describe the Feature Request

# Browser Support

The targets I would like to support are:

Chrome
Chrome for Android
Firefox
Safari for iOS

Use caniuse.com browser support page here as reference of what browsers to support.

https://caniuse.com/usage-table

### Describe the Use Case

Many people may want to view the site on Tablet, Phone and Desktop.

### Describe Preferred Solution

Media queries & specific device breakpoints.

**Comment by gbowne1** [comment]
My research indicates for a variety of tablet and phone devices, the screen size breakpoints should be 

- Tablet  {760 to 820} x {1024 to 1280}

- Phone: {300 to 500} x {600 to 1024}

  There appears to be 3 different general sizes of phones.
 
- Desktop: {more research needed}

**Comment by Rmichal503** [comment, marked as answer]
I think that 3 breakpoints will be enough
- max-width 576px (phones)
- max-width 1024px (tablets)
- and above

**Reply by gbowne1** [reply]
There should be a breakpoint for laptops too. Forgot about that

**Comment by gbowne1** [comment]
Yes I agree.
